packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -224,6 +234,13 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     }
   }
 
+  private _onRenderAddon(props): JSX.Element {
+    let { addonString } = props;
+    return (
+      <span style={ { paddingBottom: '2px' } }>{ addonString }</span>
```

Comment:
i hate this padding here, but it's needed to get default text lined up with the text in the input. If you render an icon instead it seems to be fine, so I only put the styles on this span.